projects/swimlane/ngx-ui/src/lib/components/toolbar/fixtures/toolbar.fixture.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, ViewChild, ChangeDetectionStrategy } from '@angular/core';
```

Comment:
I like that you broke this into its own file, will update my code to do the same.